### PR TITLE
Only install enum34 for python2.7 as it breaks for python36

### PR DESCRIPTION
**Pull request by bagerard**
Hi, We recently switched to pyton3.6 and the fact that enum34 gets installed automatically by rwslib is breaking a few things.
https://bitbucket.org/stoneleaf/enum34/issues/19/enum34-isnt-compatible-with-python-36

I applied a fix that is similar to a PR on grpc
https://github.com/grpc/grpc/pull/10227

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -26,6 +28,13 @@
 author = re.search("__author__ = '([^']+)'", rwsinit).group(1)
 version = re.search("__version__ = '([^']+)'", rwsinit).group(1)
 
+INSTALL_REQUIRES = ['requests', 'lxml', 'httpretty', 'six', 'click', 'faker']
```
Comment (glow-mdsol) [inline setup.py]: Could we use `enum34; python_version < '3.4'` ?

Comment (bagerard) [reply setup.py]: great suggestion, I didn't know it was possible

**Comment by glow-mdsol** [comment]
Thanks for this, can we bump the minor version by one for this as well.  Small issue with the CI, which I need to resolve before I can approve and move on.

**Comment by bagerard** [comment]
alright, version bumped to 1.2.2 :)

**Comment by glow-mdsol** [comment]
Thanks, looks good.  Just trying to sort out the travis-ci issue which is blocking merges.

**Comment by glow-mdsol** [comment]
Going to close and reopen the PR to prompt Travis CI to catch up

**Comment by glow-mdsol** [comment]
Ok, the interop of httpretty and python3 and later causes issues with the unittests (something to do with an unclosed socket).  I've made a small branch across at https://github.com/mdsol/rwslib/tree/feature/drop_python_33 to address.  Can you update the `test_rwscmd.py` per that branch and then push the commits.  It should then allow the CI to complete successfully.

**Comment by glow-mdsol** [comment]
Thanks for this.  One final change; can you remove 3.3 from the `.travis.yml`.  The setup can't deal with the versioned requirement, and TBH it's such an old release I think we can skip it

**Comment by glow-mdsol** [comment]
Also, add your handle to the `AUTHORS.rst` for the credit where due

**Comment by bagerard** [comment]
sure no problem :) its done

**Pull request merged**